### PR TITLE
fix edit project validator

### PR DESCRIPTION
**Pull request by MariyaSin**

### CODE REVIEW

File: src/ProjectService.Validation/EditProjectRequestValidator.cs [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+      #endregion
+
+      #region ShortDescription
+
+      AddFailureForPropertyIf(
+        nameof(EditProjectRequest.ShortName),
+        x => x == OperationType.Replace,
+        new Dictionary<Func<Operation<EditProjectRequest>, bool>, string>
+        {
+          { x => x.value == null || x.value.ToString().Trim().Length < 300, "Short description it too long." },
```
Comment (BelehovEgor) [inline src/ProjectService.Validation/EditProjectRequestValidator.cs:110]: typo is*

File: src/ProjectService.Validation/EditProjectRequestValidator.cs [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+      #endregion
+
+      #region ShortName
+
+      AddFailureForPropertyIf(
+        nameof(EditProjectRequest.ShortName),
+        x => x == OperationType.Replace,
+        new Dictionary<Func<Operation<EditProjectRequest>, bool>, string>
+        {
+          { x => x.value == null || x.value.ToString().Trim().Length < 30, "Short name it too long." },
```
Comment (BelehovEgor) [inline src/ProjectService.Validation/EditProjectRequestValidator.cs:98]: typo is*

File: src/ProjectService.Validation/EditProjectRequestValidator.cs [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+
+      #region Name
+
+      AddFailureForPropertyIf(
+        nameof(EditProjectRequest.Name),
+        x => x == OperationType.Replace,
+        new Dictionary<Func<Operation<EditProjectRequest>, bool>, string>
+        {
+          { x => !string.IsNullOrEmpty(x.value?.ToString().Trim()), "Name must not be empty." },
+          { x => x.value.ToString().Trim().Length < 150, "Name it too long." },
```
Comment (BelehovEgor) [inline src/ProjectService.Validation/EditProjectRequestValidator.cs:85]: typo is*

File: src/ProjectService.Validation/EditProjectRequestValidator.cs [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+      if (departmentId == Guid.Empty)
+      {
+        return false;
+      }
+
+      try
+      {
+        Response<IOperationResult<ICheckDepartmentsExistence>> response =
+          _rcDepartmentsExistence.GetResponse<IOperationResult<ICheckDepartmentsExistence>>(
+            ICheckDepartmentsExistence.CreateObj(new() { departmentId })).Result;
```
Comment (BelehovEgor) [inline src/ProjectService.Validation/EditProjectRequestValidator.cs:140]: make async

File: src/ProjectService.Validation/EditProjectRequestValidator.cs [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+          nameof(EditProjectRequest.ShortDescription)
+        });
+
+      AddСorrectOperations(nameof(EditProjectRequest.Status), new List<OperationType> { OperationType.Replace });
+      AddСorrectOperations(nameof(EditProjectRequest.DepartmentId), new List<OperationType> { OperationType.Replace });
+      AddСorrectOperations(nameof(EditProjectRequest.Name), new List<OperationType> { OperationType.Replace });
+      AddСorrectOperations(nameof(EditProjectRequest.ShortName), new List<OperationType> { OperationType.Replace });
+      AddСorrectOperations(nameof(EditProjectRequest.Description), new List<OperationType> { OperationType.Replace });
+      AddСorrectOperations(nameof(EditProjectRequest.ShortDescription), new List<OperationType> { OperationType.Replace });
+      #endregion
```
Comment (VLiamin) [inline src/ProjectService.Validation/EditProjectRequestValidator.cs:49]: Empty line?

**Review by VLiamin — APPROVED** [review]

**Pull request merged**